__tests__/tooling/operation/get-parameters-as-json-schema.test.js

```diff
@@ -1304,6 +1304,36 @@ describe('defaults', () => {
         expect(oasInstance.operation('/', 'get').getParametersAsJsonSchema()).toMatchSnapshot();
       });
 
+      it('should not add jwtDefaults if there are no matches', async () => {
+        const schema = new Oas(petstore);
+        await schema.dereference();
+        const operation = schema.operation('/pet', 'post');
+        operation.jwtDefaults = {
+          fakeParameter: {
+            id: 4,
+            name: 'Testing',
+          },
+        };
```

Comment:
This test needs to get updated to not set `jwtDefaults` on the `operation`.

Reply:
updated!